modules/nf-core/raxmlng/main.nf

```diff
@@ -7,28 +7,55 @@ process RAXMLNG {
         'biocontainers/raxml-ng:1.0.3--h32fcf60_0' }"
 
     input:
-    path alignment
+    tuple val(meta), path(alignment), val(model)
 
     output:
-    path "*.raxml.bestTree", emit: phylogeny
-    path "*.raxml.support" , optional:true, emit: phylogeny_bootstrapped
-    path "versions.yml"    , emit: versions
+    tuple val(meta), path("*.raxml.bestTree")              , emit: phylogeny
+    tuple val(meta), path("*.raxml.support"), optional:true, emit: phylogeny_bootstrapped
+    path "versions.yml"                                    , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
+    def args = task.ext.args ?: params.raxmlng_args ?: ''
+    def prefix = task.ext.prefix ?: meta.id
+    if (!meta.id) {
+        error "Input meta map does not contain 'id'. Received: ${meta}"
+    }
```

Comment:
```suggestion
```
We assume that the meta.id is present (the only part of the meta that we assume exists)